aspnet/web-forms/overview/getting-started/hands-on-labs/whats-new-in-web-forms-in-aspnet-45.md

```diff
@@ -497,7 +497,7 @@ The request validation feature in ASP.NET provides a certain level of default pr
     *Request validation disabled for the product description*
 
     > [!NOTE]
-    > In a production application, you should sanitize the HTML code entered by the user to make sure only safe HTML tags are entered (for example, there are no &lt;script&gt; tags). To do this, you can use [Microsoft Web Protection Library](http://wpl.codeplex.com/).
+    > In a production application, you should sanitize the HTML code entered by the user to make sure only safe HTML tags are entered (for example, there are no &lt;script&gt; tags). To do this, you can use [Microsoft Web Protection Library](https://www.nuget.org/packages/AntiXSS/4.3.0).
```

Comment:
Let's use the following link instead, so we're not pointing to a specific version: https://www.nuget.org/packages/AntiXSS